jenkins/parser/paser-config-unittest.py

```diff
@@ -1,5 +1,9 @@
 import json
 import os
+import re
+from subprocess import getstatusoutput
```

Comment:
@aandvalenzuela , this is good for python3 but will fail for python2. You can either do something like
```
import sys
if sys.version_info[0] == 2:
  from commands import getstatusoutput
else:
  from subprocess import getstatusoutput
```

or better to drop the python2 support and update the github yaml file to only run the test for python3

Reply:
Dropping python2 support, thanks!